client/gutenberg/extensions/README.md

```diff
@@ -36,3 +36,16 @@ Presets follow the same structure as blocks, just under `presets` folder:
         ├── view.js
         └── view.scss
 ```
+
+## Block naming conventions
+
+Blocks are registered by providing a `name` and `settings` like this:
+
+```js
+registerBlockType( 'prefix/name', { /* settings */ } );
+```
+
+Public WordPress.com blocks should use the `jetpack/` prefix, e.g. `jetpack/markdown`.
```

Comment:
Maybe remove `WordPress.com` - I'd call our blocks "Jetpack" blocks instead, so this clarifications feels unnecessary and too specific.

Reply:
Done!